portfolio/src/main/webapp/script.js

```diff
@@ -104,4 +104,28 @@ function fetchBlobstoreUrlAndShowForm() {
         messageForm.action = imageUploadUrl;
         messageForm.classList.remove('hidden');
     });
+}
+
+/** Creates a map centered in Manhattan NY with some museum locations. */
+function createMap() {
+    fetch('/museum-locations').then(response => response.json()).then((museumLocation) => {
+        const map = new google.maps.Map(
+            document.getElementById('map'),
+            {center: {lat: 40.7831, lng: -73.9712}, zoom: 12} );
```

Comment:
i think you should store this as a constant so its easy to understand that it's the Manhattan NY coords.

Reply:
ok, noted